src/eslint.js

```diff
@@ -14,7 +15,6 @@ module.exports = {
     'comma-dangle': 'error',
     'comma-spacing': 'error',
     'comma-style': 'error',
-    complexity: ['error', 4],
```

Comment:
Did you decide it's just not worth fighting this battle? Or does this just fall back to the default?

Reply:
No, it's completely disabled. You can [read this](https://github.com/manovotny/eslint-config-get-off-my-lawn/issues/95) for insight into why I disabled it.